lib/cmap/column_parser.rb

```diff
@@ -0,0 +1,34 @@
+module Cmap; class ColumnParser
```

Comment:
`VertexNameParser` might be a better name than `ColumnParser`.  I don't think we a parsing a column here.